src/Testing/ZopeTestCase/zopedoctest/functional.py

```diff
@@ -237,8 +237,9 @@ def setup_test_class(self):
 
         # If the test_class does not have a runTest method, we add
         # a dummy attribute so that TestCase construction works.
+        # runTest may get called, so set it to something that works.
         if not hasattr(test_class, 'runTest'):
-            setattr(test_class, 'runTest', None)
+            setattr(test_class, 'runTest', lambda x: None)
 
```

Comment:
When I use a checkout of Zope2 branch 2.13, do `bin/develop co AccessControl`, run buildout, and switch `AccessControl` to the branch of https://github.com/zopefoundation/AccessControl/pull/23 I see the same problem, which could be solved in the same way by fixing the dummy `runTest`.

It then already goes wrong when doing `bin/alltests -m OFS -m AccessControl`. Part of the problem may be that several `OFS` tests do this:

```

def test_suite():
    from Testing.ZopeTestCase import ZopeDocTestSuite
    return ZopeDocTestSuite()
```

So they call `ZopeDocTestSuite` without any arguments, to let it create a doctest for the current test module. Those tests *are* executed as far as I see, so it seems okay. But when I go in with a pdb, the test case inside this suite *does* have a proper `runTest` method.